src/main/resources/swagger/openapi-admin-doc.yml

```
@@ -294,6 +297,80 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+
+  /saveSettingLogin:
+    post:
+      security:
+        - basicAuth:
+            - basicAuth
+      tags:
+        - Admin Change Login
+      description: 'Check and save old Login and new login. Check new login by duplicate.
+      Check new login is duplicate in date base. Send letter with change code'
+      operationId: updateLogin
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpdateLoginRequest'
+      responses:
+        204:
+          description: Save logins and code. Send letter.
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        401:
+          description: Unauthorize
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+  /changeLogin:
+    put:
+      security:
+        - basicAuth:
+            - basicAuth
+      tags:
+        - Admin Change Login
+      description: 'Check entered code by saved code. Find admin in date base by saved login.
+      Change admin login to new login.'
```

```suggestion
      description: 'Check entered code by saved code. Find admin in date base by saved login.
        Change admin login to new login.'
```
Add tab in line 345.